packages/app/components/document/form-actions.tsx

```diff
@@ -84,31 +79,6 @@ const FormActions = ({
         }
     }
 
-    /**
```

Comment:
This was the original attempt which relied on circumventing RJSF's validation. Removing this in favor of using their validation up front and us removing all affected fields before validation runs